interfaces/matlab_experimental/Base/ThermoPhase.m

```diff
@@ -1393,6 +1393,51 @@ function display(tp)
 
         end
 
+        function set.T(tp, t)
```

Comment:
I don't think that we should allow for single setter methods, and would in general avoid requiring `input` statements.